sormas-api/src/main/resources/captions.properties

```diff
@@ -288,6 +289,8 @@ Campaign.description=Description
 Campaign.startDate=Start date
 Campaign.endDate=End date
 Campaign.creatingUser=Creating user
+Campaign.deleteReason=Delete reason
+Campaign.otherDeleteReason=Other delete reason
```

Comment:
Reason for deletion details (also change this in other places)

Reply:
changed